app/views/providers/uploaded_evidence_collections/_upload_evidence.html.erb

```diff
@@ -43,8 +60,8 @@
         <% if @mandatory_evidence_errors %>
             <p id="dropzone-mandatory-error" class='govuk-error-message dropzone-error govuk-!-margin-bottom-1'>
               <% @mandatory_evidence_errors.each do |error| %>
-                <span class="govuk-visually-hidden">Error:</span>
-                <%= error.message %>
+                  <span class="govuk-visually-hidden">Error:</span>
```

Comment:
These two lines have been indented an extra step

Reply:
Thanks for spotting that. I have made the change.